src/map/utils/battleutils.cpp

```diff
@@ -3595,6 +3607,12 @@ namespace battleutils
                 break;
         }
 
+        // Add intimidation rate from BST killer effect merits
+        if (PDefender->objtype == TYPE_PC && PDefender->GetMLevel() > 74 && PDefender->GetMJob() == JOB_BST)
```

Comment:
Similar thing here with a simpler check for GetMeritValue since it does the job checking for you, but I don't know about how exactly the merit actually works.

JP wiki and BG wiki don't mention if it's _all_ killer effects or only ones you have from traits. Does this let you intimidate humanoids (players)? probably not. I think this one will need a source or verification of some kind to figure out what exactly it does.